ml-agents/mlagents/trainers/training_status.py

```diff
@@ -12,7 +12,7 @@
 
 logger = get_logger(__name__)
 
-STATUS_FORMAT_VERSION = "0.2.0"
+STATUS_FORMAT_VERSION = "0.2.1"
```

Comment:
Why bump this? It does not seem to be used anywhere. If this is needed, you need to check for backwards compatibility at some point no?

Reply:
I think we should still tick it. We don't check it b/c there hasn't been any breaking changes with the loading of this file - if we do make a breaking change, I would think we'll have to tick to 1.0 and check. Checking if the minor versions are the same was somewhat subsumed by the mlagents version check, so we didn't add a separate check for this file. At the end of the day this versioning was intended so that Cloud could reliably use this file if needed. 

With that said I did tick it incorrectly - it should be 0.3.0 not a patch ver.

Reply:
Ok, feel free to merge when the tests pass